### ISSUE TITLE
[doc] Fix the "latest" doc alias in the web documentation

### ISSUE THREAD

**Issue by tasosalvas** [debops/debops#1465]
Related to #954

`latest` currently seems undefined in [Readthedocs](https://readthedocs.org/projects/debops/builds/). It is serving ancient doc from debops 0.8.1.

`ack "debops.org/en/latest/"` shows a bunch of usage of the link in doc, and the alias itself ain't that bad, so I'd suggest reinstating it and pointing it to the current `stable`.

We could also edit it to `master`, I guess, but a permanent link to `stable` does have utility, and it would be madness to use the versioned `stable-x.x` links in the doc.

I have no idea about implementation, all I see is its definition is not in the codebase.

These open issues were all related to misinformation spread through the bad link to some extent.
They all end with their issue being resolved, and can be closed.
* #580
* #591
* #914

**Comment by drybjed** [comment]
I enabled the "latest" version on ReadTheDocs. The target can be configured in the ReadTheDocs settings, so I pointed it to `stable-2.1` branch, and I'll switch it to the new stable release in the future. That way we don't need to have a separate `stable` branch in the repository.

In the long run, I think that absolute URLs to the DebOps documentation should be replaced by relative (internal) ones unless it's an example or something similar. That way the links point to the same branch/tag documentation of each version and there's no need to track them externally.

**Comment by tasosalvas** [comment]
Yeah haha it didn't even cross my mind that those should be internal when I wrote that... The whole links situation (plus the `../../../../global` refs) definitely needs a cleanup sprint at some point. I hoped the paragraph I added in contributing-docs would have elicited some contribs but maybe it needs some clearer instruction to become a good first time issue.

At any rate, it would break my heart to have to tell someone they've been spending hours reading the wrong doc :)
I think as damage control having that link work properly is ok for now.
